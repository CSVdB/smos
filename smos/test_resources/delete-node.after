- forest:
  - forest:
    - forest:
      - me
      entry: is
    entry: this
  entry: hello
