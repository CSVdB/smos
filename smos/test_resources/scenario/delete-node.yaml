before:
- entry: hello
  forest:
  - entry: world
    forest:
    - entry: this
      forest:
      - entry: is
        forest:
        - me
commands:
- forestMoveDown
- forestDeleteCurrentEntry
after:
- entry: hello
  forest:
  - entry: this
    forest:
    - entry: is
      forest:
      - me
