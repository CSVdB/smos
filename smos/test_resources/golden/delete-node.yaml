before:
- forest:
  - forest:
    - forest:
      - forest:
        - me
        entry: is
      entry: this
    entry: world
  entry: hello
commands:
- forestToggleCollapse
- forestMoveDown
- forestDeleteCurrentEntry
after:
- forest:
  - forest:
    - forest:
      - me
      entry: is
    entry: this
  entry: hello
